XM_Dinner/415_Combee.yaml

```diff
@@ -0,0 +1,22 @@
+species: Combee

```

Comment:
Move order, should be heal order over recover probably, very very under-powered despite the dangerous combination of defend order and healing. standard combee does something similar with more investment if you want a guide https://github.com/TwitchPlaysPokemon/pbr-movesets/blob/master/1%20Standard/415_Combee.yaml

Reply:
Might do
    - [Attack Order]
    - [Sludge Bomb]
    - [Giga Drain]
    - [Toxic]
thoughts?